### PR TITLE
[ADD] Added a project_mail_chatter module.

### PR DESCRIPTION
**Pull request by AmmarOfficewalaSerpentCS**

### CODE REVIEW

File: project_mail_chatter/views/project_view.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- © 2015 Antiun Ingeniería S.L. - Jairo Llopis
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
```
Comment (dreispt) [inline project_mail_chatter/views/project_view.xml]:
```suggestion
```

File: project_mail_chatter/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2015-2017 Tecnativa - Jairo Llopis <jairo.llopis@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Project Mail Chatter",
+    "summary": "Display Project log in a chatter view.",
```
Comment (dreispt) [inline project_mail_chatter/__manifest__.py]:
```suggestion
    "summary": "Add message chatter on the Project form.",
```

File: project_mail_chatter/readme/DESCRIPTION.rst [hunk 3]
```diff
@@ -0,0 +1 @@
+This module extends the functionality of projects to display project log in a project chat area.
```
Comment (dreispt) [inline project_mail_chatter/readme/DESCRIPTION.rst]:
```suggestion
Add message chatter on the Project form.
```

**Review by dreispt — APPROVED** [review]
LGTM, minor suggestions

**Review by dreispt — APPROVED** [review]

**Review by HaraldPanten — APPROVED** [review]

**Review by marcelsavegnago — APPROVED** [review]

**Pull request merged**